controller.js

```diff
@@ -1,8 +1,12 @@
 const endpoints = require('./endpoints.json')
+const topicsData = require('./db/data/development-data/topics')
```

Comment:
This isn't where the topicsData should be coming from. It should be coming from a model which gets the data out of the database.

The model should be querying the database and then sending the information back to the controller